src/amount.h

```diff
@@ -17,7 +17,7 @@ typedef int64_t CAmount;
 static const CAmount COIN = 100000000;
 static const CAmount CENT = 1000000;
 
-extern const std::string CURRENCY_UNIT;
```

Comment:
Is there no other "generic" or "utils" .o file that is linked with bitcoin-cli and bitcoind?  if not, then move it